### ISSUE TITLE
Test Failing because of the Uncaught exception on cy.visit

### ISSUE THREAD

**Issue by gurjeetbains** [cypress-io/cypress#5547]
Hi,

My test fails because of the Uncaught type error when I execute `cy.visit()`. My next steps execute but still the test fails.
I have used 
```js
Cypress.on('uncaught:exception', (err, runnable) => {
    // returning false here prevents Cypress from
    // failing the test
    return false
  })
```

![Cypress](https://user-images.githubusercontent.com/19879766/67917347-949b1580-fbfd-11e9-9c51-8cbc2a5e0cce.PNG)

**Comment by jennifer-shehane** [comment]
@gurjeetbains This exception is likely being thrown from a plugin included in your application. Have you tried clicking on the error and seeing where the error is being thrown to track down where it originates? 

We have one bug fix going out to catch some more of the uncaught exceptions here: https://github.com/cypress-io/cypress/issues/987

Without a reproducible example of this situation, we'll be unable to help unfortunately. Can you provide a complete set of code we can run to reproduce?

**Comment by Bkucera** [comment]
I believe we immediately fail the test instead of routing the AUT error through 'uncaught:exception', which is a bug. Closely related to work done in #5249

**Comment by gurjeetbains** [comment]
![image](https://user-images.githubusercontent.com/19879766/67978607-6231fc80-fc7f-11e9-9426-b4230aba11e8.png)
I can't share the code, but this is the complete error we get when we click on the error @jennifer-shehane

**Comment by brian-mann** [comment]
In the last release `3.6.0` we fixed two cases of this happening, and this is another situation where we missed and we can fix quickly and get this out in `3.6.1`.

**Comment by gurjeetbains** [comment]
I upgraded cypress and it is fixed now. Thanks for the help @brian-mann @jennifer-shehane @Bkucera  Closing the issue now. :)
